static/usage/modal/inline/basic/angular/app_component_html.md

```diff
@@ -0,0 +1,30 @@
+```html
+<ion-app>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>Inline Modal</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content class="ion-padding">
+    <ion-button id="open-modal" expand="block">Open</ion-button>
+    <p>{{ message }}</p>
+    <ion-modal trigger="open-modal" (willDismiss)="onWillDismiss($event)">
+      <ng-template>
+        <ion-header>
+          <ion-toolbar>
+            <ion-button slot="start" fill="clear" (click)="cancel()" color="medium">Cancel</ion-button>
```

Comment:
Optional: The grayed out Cancel button leads me to believe it is supposed to be disabled, but it is not. Maybe a different color/font weight would be good? On iOS the Confirm button is usually a heavier font weight than the Cancel button.